src/components/Card/Card.tsx

```diff
@@ -1,15 +1,16 @@
 import React, { useState, useRef } from 'react';
 import './Card.scss';
-import { CardTypes } from '../../types/CardTypes';
-import editBtn from '../../img/edit.svg';
-import { useAppDispatch } from '../../app/hooks';
-import { moveCard, removeCard, renameCard } from '../../slices/columnSlice';
-import { ModalRenameCard } from '../ModalRenameCard/ModalRenameCard';
-import { CardModal } from '../CardModal/CardModal';
-import { getColorBasedOnDate } from '../../helpers/functions/getColorBasedOnDate';
-import { formatDate } from '../../helpers/functions/formatDate';
+import editBtn from 'img/edit.svg';
+import { CardTypes } from 'types/CardTypes';
+import { useAppDispatch } from 'store/hooks';
 import { useDrag, useDrop } from 'react-dnd';
-import { updateTextAreaHeight } from '../../helpers/functions/updateTextAreaHeight';
+import { moveCard, removeCard, renameCard } from 'slices/columnSlice';
+import { updateTextAreaHeight } from 'helpers/functions/updateTextAreaHeight';
+import { ModalRenameCard } from 'components/ModalRenameCard/ModalRenameCard';
```

Comment:
personally I see name of this component as RenameCardModal for better readability. What do you think?

Reply:
I`m absolutly agree.